sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -180,7 +181,11 @@ def _get_request_data(self, request):
         """
         if request.files:
             form_data = aiohttp.FormData(request.data or {})
-            for form_file, data in request.files.items():
+            if isinstance(request.files, Mapping):
```

Comment:
why is this code specifically in `aiohttp`? Why not in the shared_pipeline_transport_helper

Reply:
`requests` can accept dict[] or list[tuple] while `aiohttp` can only accept list[tuple].
I create a helper method and put it in `_pipeline_transport_rest_shared`